src/app/pages/signin/store/signin.store.ts

```diff
@@ -240,8 +248,14 @@ export class SigninStore extends ComponentStore<SigninState> {
   }
 
   private handleLoginWithToken(): Observable<LoginResult> {
```

Comment:
Method gets a bit convoluted with all the conditions. How about introducing a new method for logging in with query token and either calling `handleLoginWithToken` or that new method depending on whether query parameter set.